src/main/java/com/snowflake/kafka/connector/SchematizationUtils.java

```diff
@@ -55,7 +63,9 @@ public static Map<String, String> getAvroSchemaFromSchemaRegistryClient(
     try {
       schemaMetadata = schemaRegistry.getLatestSchemaMetadata(subjectName);
     } catch (IOException | RestClientException e) {
-      throw SnowflakeErrors.ERROR_0012.getException(e);
+      // suppress the excpetion and return empty map to indicate a failure in fetching schemas
+      LOGGER.debug("Schema with subject '{}' not found in schema registry", subjectName);
+      return new HashMap<>();
```

Comment:
return Collections.emptyMap() to indicate immutability
https://stackoverflow.com/questions/14846920/collections-emptymap-vs-new-hashmap